client/app/src/main/java/com/microsoft/research/karya/data/local/daos/MicroTaskAssignmentDao.kt

```diff
@@ -40,4 +40,12 @@ interface MicroTaskAssignmentDao : BasicDao<MicroTaskAssignmentRecord> {
     insertForUpsert(records)
     updateForUpsert(records)
   }
+
+  @Query("SELECT MAX(created_at) FROM microtask_assignment WHERE status != :status")
+  suspend fun getNewAssignmentsFromTime(status: MicrotaskAssignmentStatus = MicrotaskAssignmentStatus.VERIFIED): String?
+
+  @Query("SELECT MAX(created_at) FROM microtask_assignment WHERE status == :status")
+  suspend fun getNewVerifiedAssignmentsFromTime(
+    status: MicrotaskAssignmentStatus = MicrotaskAssignmentStatus.VERIFIED
+  ): String?
```

Comment:
Add a `TODO` here to also add a check for `worker_id`

Reply:
Sure

Reply:
I am instead passing the worker_id and implementing it